packages/msw-addon/package.json

```diff
@@ -36,8 +36,8 @@
     "is-node-process": "^1.0.1"
   },
   "devDependencies": {
-    "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.0.0"
+    "tsc": "^2.0.3",
```

Comment:
this look like it was added by mistake?

Reply:
It's used to build the library, why was it a mistake?

Reply:
the typescript compiler `tsc` is part of the `typescript` package -- the `tsc` package is, apparently, "A deprecated release of the TypeScript compiler" -- see `npm info tsc`.

Reply:
Today I learned! Thanks for sharing that info. Fixed!